src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -341,6 +331,32 @@ void AreaTrigger::HandleUnitEnterExit(std::list<Unit*> newTargetList)
     }
 }
 
+void AreaTrigger::UpdatePolygonOrientation()
+{
+    float newOrientation = GetOrientation();
+
+    // No need to recalculate, orientation didn't changed
```

Comment:
Damn, saw it after commit, hoped i would be able to fix it before anyone notice x)